caluma/caluma_core/visibilities.py

```diff
@@ -79,6 +80,8 @@ def filter_queryset(self, node, queryset, info):
         result_queryset = None
         for visibility_class in self.visibility_classes:
             class_result = visibility_class().filter_queryset(node, queryset, info)
+            if isinstance(class_result, Manager):
```

Comment:
Instead of checking the type, I'd suggest just calling `.all()` on the result. This method is supported by both managers and querysets.

Reply:
Fixed.